app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::BaseController < ActionController::Base
+  protect_from_forgery with: :exception
+
```

Comment:
remove this line

Reply:
em có up lại font và đã giải quyết được rồi ạ